src/controllers/project.js

```diff
@@ -0,0 +1,74 @@
+const Project = require('../models/project');
+const User = require('../models/user');
+
+const isApplicableAmount = (amount, collected, requested) =>
```

Comment:
yeah please add curly brackets and the return statement here

Reply:
eslint does not accept that since the default return can be placed after => directly and gives error otherwise

Reply:
![image](https://user-images.githubusercontent.com/14183741/152514945-51248c17-fc9e-4b5d-b7fd-1aaf99b78d26.png)